sim/warlock/demonology/hand_of_guldan.go

```diff
@@ -8,52 +8,39 @@ import (
 )
 
 func (demonology *DemonologyWarlock) CurseOfGuldanDebuffAura(target *core.Unit) *core.Aura {
+	// TODO: the talent tooltip says this applies to "any Warlock demons". It's unclear if this means
+	// any warlock pet or just the ones belonging to the caster
 	return target.GetOrRegisterAura(core.Aura{
-		Label:    "CurseOfGuldan-" + demonology.Label,
+		Label:    "Curse of Guldan-" + demonology.Label,
 		ActionID: core.ActionID{SpellID: 86000},
 		Duration: 15 * time.Second,
 		OnGain: func(aura *core.Aura, sim *core.Simulation) {
-			//TODO: Implement Crit rating for pet vs this target only
+			for _, pet := range demonology.Pets {
+				pet.AttackTables[aura.Unit.UnitIndex].BonusCritRating += 0.1
```

Comment:
Should this not add `core.CritRatingPerCritChance * critBonus` instead of just 0.1

Reply:
No; for whatever reason and despite the name this variable modifies the crit chance directly, i.e. 0.1 = 10% crit.